Vagrantfile

```diff
@@ -0,0 +1,32 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"
+
+  config.vm.provision "shell", privileged: false, inline: <<-SHELL
+    sudo apt-get update
+	sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+	curl https://pyenv.run | bash
+	echo 'export PATH="$HOME/.pyenv/bin:$PATH"' >> ~/.profile
+	echo 'eval "$(pyenv init -)"' >> ~/.profile
+	echo 'eval "$(pyenv virtualenv-init -)"' >> ~/.profile
+	source ~/.profile
+	export PATH="/home/vagrant/.pyenv/bin:$PATH"
+	eval "$(pyenv init -)"
+	eval "$(pyenv virtualenv-init -)"
```

Comment:
You shouldn't need to repeat these three lines now that you are updating ~/.profile and running `source ~/.profile`